mmdet/core/visualization/image.py

```diff
@@ -169,3 +173,118 @@ def imshow_det_bboxes(img,
+      gt_text_color (str or tuple(int) or :obj:`Color`):Color of texts.
+           The tuple of color should be in BGR order. Default: (255, 102, 61)
+      gt_mask_color (None or str or tuple(int) or :obj:`Color`):
+           Color of masks. The tuple of color should be in BGR order.
+           Default: (255, 102, 61)
+      det_bbox_color (str or tuple(int) or :obj:`Color`):Color of bbox lines.
+           The tuple of color should be in BGR order. Default: (72, 101, 241)
+      det_text_color (str or tuple(int) or :obj:`Color`):Color of texts.
+           The tuple of color should be in BGR order. Default: (72, 101, 241)
+      det_mask_color (None or str or tuple(int) or :obj:`Color`):
+           Color of masks. The tuple of color should be in BGR order.
+           Default: (72, 101, 241)
+      thickness (int): Thickness of lines. Default: 2
+      font_size (int): Font size of texts. Default: 13
+      win_name (str): The window name. Default: ''
+      fig_size (tuple): Figure size of the pyplot figure. Default: (15, 10)
+      show (bool): Whether to show the image. Default: True
+      wait_time (float): Value of waitKey param. Default: 0.
+      out_file (str or None): The filename to write the image. Default: None
+    """
+    assert 'gt_bboxes' in annotation
+    assert 'gt_labels' in annotation
+    assert isinstance(result, (tuple, list))
+
+    gt_masks = annotation.get('gt_masks', None)
+    if gt_masks is not None:
+        if isinstance(gt_masks, (BitmapMasks, PolygonMasks)):
+            gt_masks = gt_masks.to_ndarray()
+        elif isinstance(gt_masks, torch.Tensor):
+            gt_masks = gt_masks.detach().cpu().numpy()
+        elif not isinstance(gt_masks, np.ndarray):
+            warnings.warn(f'Unsupported {type(gt_masks)} data type')
+            gt_masks = None
+
+    img = mmcv.imread(img)
+    img = imshow_det_bboxes(
+        img,
+        annotation['gt_bboxes'],
+        annotation['gt_labels'],
+        gt_masks,
+        class_names=class_names,
+        bbox_color=gt_bbox_color,
+        text_color=gt_text_color,
+        mask_color=gt_mask_color,
+        thickness=thickness,
+        font_size=font_size,
+        win_name=win_name,
+        fig_size=fig_size,
+        show=False)
+    if isinstance(result, tuple):
+        bbox_result, segm_result = result
+        if isinstance(segm_result, tuple):
+            segm_result = segm_result[0]  # ms rcnn
+    else:
+        bbox_result, segm_result = result, None
+    bboxes = np.vstack(bbox_result)
+    labels = [
+        np.full(bbox.shape[0], i, dtype=np.int32)
+        for i, bbox in enumerate(bbox_result)
+    ]
+    labels = np.concatenate(labels)
+    segms = None
+    if segm_result is not None and len(labels) > 0:  # non empty
+        segms = mmcv.concat_list(segm_result)
+        segms = mask_util.decode(segms)
+        segms = segms.transpose(2, 0, 1)
+    imshow_det_bboxes(
```

Comment:
This function has return, so the return of imshow_det_bboxes should also be returned by this function.